Cargo.toml

```diff
@@ -19,7 +19,7 @@ ipc = { version = "0.1.0", path = "ipc" }
 notify-rust = "4"
 chrono = { version = "0.4", features = ['serde'] }
 fern = "0.6.1"
-iced = { version = "0.13.0-dev", features = [
+iced = { version = "0.13.1", features = [
```

Comment:
Add `0.14.0-dev` here instead, as we always target latest master commit.

Reply:
@casperstorm I'm getting these errors when building against `0.14.0-dev` and latest master commit, looks like the color struct has moved, I'll do some tweaking!

```
note: `iced_core::color::Color` is defined in crate `iced_core`
   --> /home/quaff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/iced_core-0.13.2/src/color.rs:5:1
    |
5   | pub struct Color {
    | ^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `iced_core` are being used?
```